slovakqwerty/manifest.json

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2015 Matus Marcin (matusmarcin.com) All rights reserved.
```

Comment:
Please change the copyright to one of the following:
1) Copyright 2015 Google Inc. All rights reserved.
or
2) Copyright 2015 the Extra Keyboard project Authors. All rights reserved.